giantswarm/circleci-exporter
# Histogram & gauge metrics

### How to test

1. Run the operator
```
kubeconfig=`cat $(kind get kubeconfig-path --name="kind")` && go run main.go daemon --service.circleci.auth.token=xyz | luigi
```
2. Run prometheus
```
cat prom.yml
scrape_configs:
  - job_name:       'example-random'

    static_configs:
      - targets: ['localhost:8000']
        labels:
          group: 'production'

./prometheus --config.file=prom.yml
```

3. Check it works by connecting http://localhost:9090/graph
